Track generated OpenAPI definition as JSON in the source tree (#2987) (#2997)
DataBiosphere/azul: README.md

The Azul project contains the components that together serve as the backend to
Boardwalk, a web application for browsing genomic data sets.

> # Table of Contents

Reviewer: Isn't deleting this covered in another PR or ticket somewhere?

It looks like it is, but it's showing up here because this is chained on that PR? Make sure you set the proper base branch, labels, etc. if I'm correct and this should be a chain.

Author: Its a freebie, the commit issue is referenced in the title. The base should be develop. Is there something I messed up? I thought the blocking relation would be appropriate.

Reviewer: Sorry, my mistake. I thought this was already in a separate PR. With the new Zenhub changes you could try linking the freebie as an experiment, but that's not approved process.

The script serves the Swagger editor locally at a URL where your current version
of the API documentation is visible. Change the docs in `azul/service/app.py`,
save, refresh the page, and your changes will appear immediately.

> ## 13.2 Tracking changes to the OpenAPI definition document

Reviewer 2: ```suggestion
## 13.2 Tracking changes to the OpenAPI definition
```

Author: I'll change this, but i think the "official" term is document. 
https://swagger.io/specification/#definitions

> Changes to the OpenAPI document are tracked. When making changes that affect the

Reviewer 2: ```suggestion
Changes to the OpenAPI definition are tracked in the source tree. When making changes that affect the
```

> document, run:

Reviewer 2: ```suggestion
definition, run:
```

```
make openapi
```

> and commit any modifications to `openapi.json` file. Failure to do so will 
> result in a CI pipeline failure.

Reviewer 2: ```suggestion
and commit any modifications to the `openapi.json` file. Failure to do so will 
break continuous integration during `make check_clean`.
```
